core/src/main/java/nl/nn/adapterframework/configuration/filters/Stub4TesttoolFilter.java

```diff
@@ -0,0 +1,64 @@
+/*
+   Copyright 2021 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.configuration.filters;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.sax.TransformerHandler;
+
+import org.xml.sax.ContentHandler;
+
+import nl.nn.adapterframework.configuration.Configuration;
+import nl.nn.adapterframework.configuration.ConfigurationUtils;
+import nl.nn.adapterframework.core.Resource;
+import nl.nn.adapterframework.util.TransformerPool;
+import nl.nn.adapterframework.util.XmlUtils;
+import nl.nn.adapterframework.xml.TransformerFilter;
+
+/**
+ * The Stub4TesttoolFilter is used to stub configurations during parsing
+ */
+public class Stub4TesttoolFilter extends TransformerFilter {
+	private Stub4TesttoolFilter(TransformerHandler transformerHandler, ContentHandler handler) {
+		super(null, transformerHandler, null, null, false, handler);
+	}
+	
+	/**
+	 * Get the contenthandler to stub configurations
+	 * If stubbing is disabled, the input ContentHandler is returned as-is
+	 */
+	public static ContentHandler getStub4TesttoolContentHandler(ContentHandler resolver, Properties properties) throws IOException, TransformerConfigurationException {
+		if (Boolean.parseBoolean(properties.getProperty(ConfigurationUtils.STUB4TESTTOOL_CONFIGURATION_KEY,"false"))) {
+			Resource xslt = Resource.getResource(ConfigurationUtils.STUB4TESTTOOL_XSLT);
+			TransformerPool tp = TransformerPool.getInstance(xslt);
+			
+			Stub4TesttoolFilter filter = new Stub4TesttoolFilter(tp.getTransformerHandler(), resolver);
```

Comment:
Hier kan/moet je tp.getTransformerFilter() gebruiken.
Stub4TesttoolFilter hoeft dan geen eigen class te zijn; getStub4TesttoolContentHandler kan een method in ConfigurationUtils worden